src/Bicep.Cli/Commands/RootCommand.cs

```diff
@@ -101,6 +101,26 @@ private void PrintHelp()
       bicep decompile file.json --outdir dir1
       bicep decompile file.json --force
       bicep decompile file.json --outfile file.bicep
+
+  {exeName} generate-params [options] <file>
```

Comment:
Documentation should indicate what happens if the file already exists.

Reply:
created a commit to update the help text